fbw-a380x/src/systems/instruments/src/PFD/AltitudeIndicator.tsx

```diff
@@ -781,18 +782,16 @@ class AltimeterIndicator extends DisplayComponent<AltimeterIndicatorProps> {
   private handleBlink() {
     if (this.mode.get() === 'STD') {
       if (
-        this.flightPhase > 3 &&
-        this.transLvlAr.isNormalOperation() &&
-        100 * this.transLvlAr.value > this.props.altitude.get()
+        this.fmgcFlightPhase > FmgcFlightPhase.Cruise &&
+        100 * (this.transLvlAr.isNormalOperation() ? this.transLvlAr.value : 25) > this.props.altitude.get()
```

Comment:
No default for trans level. It should not blink if the trans level is NCD or FW.

Reply:
roger, reverting to the previous logic